getting-started/data-node-authentication.md

```diff
@@ -0,0 +1,70 @@
+# Data Node Authentication
+
+In a multi-node environment, it is necessary to ensure that the access
+node have the necessary roles and permissions on all the data nodes
+where distributed hypertables are placed.
+
+This is critical since when TimescaleDB is executing a query or
+command it will use the same role on the data nodes for the
+fragments. This means that the same role need to exist on the data
+nodes and authenticate to them.
+
+In a typical setup, you will deploy the distributed hypertable in a
+database and use the same database on all the data nodes. This means
+that we need to ensure that a user that is connected to the access
```

Comment:
```suggestion
that you need to ensure that a user that is connected to the access
```
to stay consistent

Reply:
Agree. There were another one that I also replaced with a "you".